Formula/root.rb

```diff
@@ -53,7 +57,7 @@ def install
       -Dmysql=OFF
       -Droofit=ON
       -Dssl=ON
-      -Dtbb=OFF
```

Comment:
Should we `-Dtbb=ON` or is that implied by `-Dimt=ON`?

Reply:
There is no `tbb` option, `imt` is the right name for it (for implicit multi-threading).